integration/tests/cook/test_basic.py

```diff
@@ -0,0 +1,303 @@
+import json
+import requests
+from retrying import retry
+import time
+import unittest
+import uuid
+
+class CookTest(unittest.TestCase):
+    _multiprocess_can_split_ = True
+
+    @retry(stop_max_delay=120000, wait_fixed=1000)
+    def wait_for_job(self, job_id, status):
+        job = self.session.get('%s/rawscheduler?job=%s' % (self.cook_url, job_id))
+        self.assertEqual(200, job.status_code)
+        job = job.json()[0]
+        self.assertEqual(status, job['status'])
+        return job
+
+    def minimal_job(self, **kwargs):
+        job = {
+            'max_retries': 1,
+            'mem': 10,
+            'cpus': 0.1,
+            'uuid': str(uuid.uuid4()),
+            'command': 'echo hello',
+            'name': 'echo',
+            'priority': 1
+        }
+        job.update(kwargs)
+        return job
+
+    def minimal_group(self, **kwargs):
+        group = {"uuid" : str(uuid.uuid4())}
+        group.update(kwargs)
+        return group
+
+    def setUp(self):
+        self.cook_url = 'http://localhost:12321'
```

Comment:
Is there a standard environment variable for referring to the cook scheduler URL?  Maybe just use COOK_SCHEDULER_URL.  Use http://localhost:12321 if it's not defined.  Also maybe add some helpful print statements for the user.